src/modules/ROOT/pages/changelog/202303/02.json

```diff
@@ -0,0 +1,233 @@
+{
+  "meta": {
+    "releasedOn": "2023.06.08"
+  },
+  "added": [
+    {
+      "what": [
+        "option for transparent color in the color picker."
+      ],
+      "why": [
+        "to give users more configuration options."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1211"
+      ]
+    },
+    {
+      "what": [
+        "the option to set a record list filter as a preset filter for a user."
+      ],
+      "why": [
+        "to give the user the ability to filter items in a record list before the list is loaded."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1227/commits/aa28be29cba69efce55028fa08c3a6c9a45ccc3d"
+      ]
+    },
+    {
+      "what": [
+        "menu with 'more' options (delete, un-delete and permissions) to resources in {APP_NAME_COMPOSE}, {APP_NAME_ADMIN}, {APP_NAME_REPORTER}, {APP_NAME_WORKFLOW}."
+      ],
+      "why": [
+        "so the user doesn't have to scroll horizontally to access all resource button/options."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/46f5455bd0ae67b9de2159f9d04b054a711832e1"
+      ]
+    },
+    {
+      "what": [
+        "page layouts to namespace export resources."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/4ad121747d136f76aa8701f07840c9b75c4aef37"
+      ]
+    }
+  ],
+  "changed": [
+    {
+      "what": [
+        "{APP_NAME_ADMIN} role and user picker with vue-select."
+      ],
+      "why": [
+        "to be consistent with the rest of the pickers used in {PRODUCT_NAME}."
+      ],
+      "how": [
+        "by adding vue-select to role and member picker components."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1194"
+      ]
+    },
+    {
+      "what": [
+        "the implementation for reminders in {APP_NAME_ADMIN}."
+      ],
+      "why": [
+        "to be consistent with the rest of the {PRODUCT_NAME} UI and improve the UE."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1239"
+      ]
+    },
+    {
+      "what": [
+        "the feedback email domain from `crust.tech` to `cortezaproject.org`."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/0072a4bf4004f667bc98c6543392c265e4c3991e"
+      ]
+    },
+    {
+      "what": [
+        "discarded the snoozed flag when the reminded at flag is changed in reminders."
```

Comment:
Describe the behaviour from the user-facing perspective, not so much from an implementational perspective.

If a user does this, then that happens; keep it abstract and not too technical